src/main/java/software/amazon/qldb/exceptions/StatementException.java

```diff
@@ -0,0 +1,41 @@
+package software.amazon.qldb.exceptions;
+
+import software.amazon.awssdk.services.qldbsessionv2.model.QldbSessionV2Exception;
+import software.amazon.awssdk.services.qldbsessionv2.model.StatementError;
+
+/**
+ * Exception used to represent all statement related failures that is returned from QLDB service side.
+ *
+ */
+public class StatementException extends QldbSessionV2Exception {
```

Comment:
Why does it extend the sdk exception? I'm not sure we should do this, since we are throwing something from outside the sdk.

Reply:
Should make it extend the QldbDriverException probably and we also don't want QldbDriverException extend QldbSessionV2Exception, correct? https://github.com/awslabs/amazon-qldb-driver-java/blob/master/src/main/java/software/amazon/qldb/exceptions/QldbDriverException.java#L25